images/scripts/fedora.setup

```diff
@@ -40,6 +40,7 @@ openssl \
 PackageKit \
 pcp \
 pcp-libs \
+python3-pip \
```

Comment:
I'm not a fan, but let's discuss in https://github.com/cockpit-project/cockpit/pull/17922/files#r1024140524 . This encourages users to do the wrong thing -- we strive for projects to prepare VM and run tests entirely offline, so that it becomes reproducible.

Reply:
`pip` is only ever being used to install a wheel that was generated locally.  This use is really more like `rpm` than it is `dnf`.

Reply:
Maybe, but I'd still find it a bit more elegant/robust (and way easier) to do this with a mere directory copy, instead of (1) rebuilding *all* our images first, and (2) suggesting that it was okay to run `pip` in general on our images. But as I said, let's discuss in the cockpit PR.